File: Challenges/Coding World Cup/2023/Day 25 - IND vs ENG - Maximize Team Strength.py
import math


def maximumPower(n: int, arr: list) -> str:
    # Write your code here.
    arr.sort()
    ans = float("-inf")
    for i in range(1, len(arr)):
        ans = max(ans, arr[i - 1] * arr[i])

    return str(ans)

File: Challenges/Coding World Cup/2023/Day 25 - IND vs ENG - Maximize Team Strength.java
import java.util.Arrays;

public class Solution {
    public static long maximumPower(int n, int[] a) {
        Arrays.sort(a);
        long ans = Integer.MIN_VALUE;
        for (int i = 1; i < n; i++) {
            ans = Math.max(ans, (long) a[i - 1] * a[i]);
        }
        return ans;
    }
}